.github/workflows/deploy_docs.yml

```diff
@@ -16,7 +16,6 @@ jobs:
 
       - name: Build
         run: |
-          yarn bootstrap
```

Comment:
Looks like this should use `yarn install` like in `.github/workflows/check_docs.yml`.